metacat-main/src/main/java/com/netflix/metacat/main/api/v1/TagController.java

```diff
@@ -167,11 +171,20 @@ public TagController(
         @ApiParam(value = "Prefix of the database name")
         @Nullable @RequestParam(name = "databaseName", required = false) final String databaseName,
         @ApiParam(value = "Prefix of the table name")
-        @Nullable @RequestParam(name = "tableName", required = false) final String tableName
+        @Nullable @RequestParam(name = "tableName", required = false) final String tableName,
+        @ApiParam(value = "Qualified name type")
+        @Nullable
+        @RequestParam(name = "type", required = false) final QualifiedName.Type type
```

Comment:
Why is the type different from that in the API interface?

Reply:
Will change the client to use Type